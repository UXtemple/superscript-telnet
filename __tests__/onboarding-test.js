const chat = require('./chat.js')

const messages = {
  'should onboard designers': [
    'onboarding',
    'tom',
    'designer',
  ],
  // 'should onboard developers': [
  //   'onboarding',
  //   'dario',
  //   'developer',
  // ]
}

describe('onboarding', () => {
  Object.keys(messages).forEach(title => {
    it(title, async () => {
      const conversations = await chat(messages[title])
      expect(conversations).toMatchSnapshot()
    })
  })
})
