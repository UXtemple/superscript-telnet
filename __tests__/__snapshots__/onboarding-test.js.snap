exports[`onboarding should onboard designers 1`] = `Promise {}`;

exports[`onboarding should onboard developers 1`] = `Promise {}`;
